docs/en/setup/backend/backend-fetcher.md

```diff
@@ -85,3 +85,43 @@ that adds the procedure from raw data to minute rate.
 
 When you specify `avgHistogram` and `avgHistogramPercentile`, the source should be the type of `histogram`. A counterFunction
 is also needed due to the `bucket`, `sum` and `count` of histogram are counters.
+
+## Kafka Fetcher
+
+Kafka Fetcher pulls messages from Kafka Broker(s) what is the Agent delivered. Check the agent documentation about the details. Typically Tracing Segments, Service/Instance properties, JVM Metrics, and Meter system data are supported.  Kafka Fetcher can work with gRPC/HTTP Receivers at the same time for adopting different transport protocols.
+
+Kafka Fetcher is disabled in default, and we configure as following to enable.
+
+```yaml
+kafka-fetcher:
+  selector: ${SW_KAFKA_FETCHER:default}
+  default:
+    bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
+```
+
+`skywalking-segments`, `skywalking-metrics`, `skywalking-profile`, `skywalking-managements` and `skywalking-meters` topics are required by `kafka-fetcher`.
+If they do not exist, Kafka Fetcher will create them in default. Also, you can create them by yourself before the OAP server started.
+
+When using the OAP server automatical creation mechanism, you could modify the number of partitions and replications of the topics through the following configurations:
+
+```yaml
+kafka-fetcher:
+  selector: ${SW_KAFKA_FETCHER:default}
+  default:
+    bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
+    partitions: 1
+    replicationFactor: 1
+```
+
+In the cluster mode, it should be noted that the number of topic partitions is the same as that of OAP servers. In other words, each OAP can only subscribe the specified partition. Therefore, the 'serverId' must be configured to specify which partition to consume.
+
+Kafka Fetcher allows to configure all the Kafka producers listed [here](http://kafka.apache.org/24/documentation.html#consumerconfigs) in property `kafkaConsumerConfig`. Such as:
+```yaml
+kafka-fetcher:
+  selector: ${SW_KAFKA_FETCHER:default}
+  default:
+    bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
+    kafkaConsumerConfig:
```

Comment:
I think we should highlight `partitions` value here, but with `-1` as default value.